src/openrct2/ride/Ride.cpp

```diff
@@ -7016,6 +7030,8 @@ void sub_6CB945(Ride* ride)
         CoordsXY location = { locationCoords.x * 32, locationCoords.y * 32 };
 
         TileElement* tileElement = map_get_first_element_at(location.x >> 5, location.y >> 5);
+        if (tileElement == nullptr)
+            return;
```

Comment:
Its unlikely that there is anything else useful in the vector but there might be so use `continue;`